evennia/utils/tests/test_containers.py

```diff
@@ -28,12 +47,22 @@ def test_init_with_no_scripts(self):
     self.assertEqual(len(gsc.loaded_data), 0)
 
   @override_settings(GLOBAL_SCRIPTS={'script_name': {}})
-  def test_init_with_typeclassless_script(self):
+  def test_init_with_typeclassless_script_accepts_it_asis(self):
 
     gsc = containers.GlobalScriptContainer()
 
     self.assertEqual(len(gsc.loaded_data), 1)
     self.assertIn('script_name', gsc.loaded_data)
+    self.assertNotIn("typeclass", gsc.loaded_data["script_name"])
```

Comment:
Maybe it should have defaulted to `settings.BASE_SCRIPT_TYPECLASS`? Surely, there is no functionality to fall back to anything.